yarn.lock

```diff
@@ -98,6 +98,13 @@
   dependencies:
```

Comment:
Weird, this file is on `.gitignore`.

Reply:
yarn.lock should not be on gitignore, it provides a consistent dev environment for everyone